src/server/controller/CurvytronController.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Curvytron Controller
+ *
+ * @param {Collection} clients
+ */
+function CurvytronController(clients) {
+    // create socket for all clients
+    this.clientGroup = new SocketGroup(clients);
```

Comment:
Is this controller really necessary? How is it different from the RoomsController?